src/utils/events/event-manager.js

```diff
@@ -1,6 +1,6 @@
 import WheelInput from './wheel-input';
 import MoveInput from './move-input';
-import {isBrowser} from '../globals';
+import {isBrowser} from '../../lib/utils/globals';
```

Comment:
Some of these utils should be moved up a level if they are no longer used only inside lib.